redis_inventory.py

```diff
@@ -58,12 +58,7 @@ def put_product(self, id, info):
 
 
   def delete_product(self, product_id):
-    if self.redis.exists(product_id):
-      self.redis.delete(product_id)
-      return True
-    else:
-      return False
-
+    self.redis.delete(product_id)
```

Comment:
Why did you delete the existence check..? It make sense to remove return value but I think we need to keep existence check just in case that customer tries to delete non-existing things.